libs/cvr-fixture-generator/src/generate_cvrs.ts

```diff
@@ -383,3 +400,19 @@ export function* generateCvrs({
     }
   }
 }
+
+/**
+ * Retroactively populates the hashes on a cast vote record image data object
+ */
+export function populateImageAndLayoutFileHashes(
```

Comment:
I know this is a bit hacky 🙈

It was the easiest way to get the job done without significantly refactoring this code, which I didn't want to spend time on given my eventual plan to fully overhaul CVR fixture generation to use actual CVR export functions.

I am taking a quick shortcut in that I'm only populating hashes on the `CVR.ImageData` objects stored at `BallotImage`, not the ones stored at `WriteInImage`, since we don't actually use those and it would notably complicate this function.